README.md

```diff
@@ -3,13 +3,13 @@
 Rotate files with timestamp postfix.
 
 ## Rotating by Lines #
-  
+
```

Comment:
What's going on here? Replacing \n by \n\r? We should not have this in the diff.